tests/test_compute_funs.py

```diff
@@ -1683,3 +1683,17 @@ def test_surface_equilibrium_geometry():
                 rtol=3e-13,
                 atol=1e-13,
             )
+
+
+@pytest.mark.unit
+def test_basis_kwarg():
+    """Test that we don't change basis to xyz erroneously assuming input is in rpz."""
+    eq = get("W7-X")
+    names = ["b", "e^rho", "e_theta_PEST", "phi"]
+    data_rpz = eq.compute(names)
```

Comment:
Just to be extra safe and future-proof, let's explicitly input `basis="rpz"` here instead of assuming the default.